### PR TITLE
(#4305) - Temporarily revert to old travis infra

### PR DESCRIPTION
**Pull request by daleharvey**

### CODE REVIEW

**Comment by daleharvey** [comment]
We obviously want to get on the new infrastructure, but we are losing the coverage reporting and it looks like it may be causing other version issue, so lets revert it temporarily, get the tree nice and green again then see about turning it back on

**Comment by nolanlawson** [comment]
Yes absolutely; I was too hasty about migrating to the new system. Our tree is so red right now it ain't funny.

File: .travis.yml [hunk 1]
```diff
@@ -9,12 +9,12 @@ services:
...
+# sudo: false
 
-cache:
-  directories:
-    - node_modules
-    - /tmp/phantomjs
+# cache:
+#   directories:
+#     - node_modules
+#     - /tmp/phantomjs
```
Comment (nolanlawson) [inline .travis.yml]: cache might still be good regardless, but yeah, let's turn it on after we get everything green

**Comment by nolanlawson** [comment]
I believe this will break couchdb-master (already broken) unless the apt-get works in sudoless mode, but +1 when green regardless.

**Comment by nolanlawson** [comment]
lol, coverage is still failing with `Not a valid object name`. We can't win, Dale.

**Comment by nolanlawson** [comment]
Green, merged.

**Pull request merged**